component/wallet-cli/pkg/walletrunner/wallet_runner_oidc4ci.go

```diff
@@ -229,6 +236,186 @@ func (s *Service) RunOIDC4CI(config *OIDC4CIConfig, hooks *Hooks) error {
 	return nil
 }
 
+func extractWalletInitiatedFlowParams(scopes []string) *walletInitiatedFlowParams {
+	for _, scope := range scopes {
+		chunks := strings.Split(scope, "||")
```

Comment:
Pass only issuerURL. Do not need to crete combined string that includes claimsEndpoint and credentialTemplateID